src/components/AboutTeam.tsx

```diff
@@ -0,0 +1,213 @@
+import React, { useState, useEffect } from 'react';
+import { Helmet } from 'react-helmet';
+import ProfileCard from './ProfileCard';
+import RectangleSVG from '../static/images/rectangle.svg';
+
+//image imports
+import ConnorChongImg from '../static/images/connorChong Small.png';
+import carolynCaiImg from '../static/images/carolynCai Small.png';
+import coraWenImg from '../static/images/coraWen Small.png';
+import emmaTsaiImg from'../static/images/emmaTsai Small.png';
+import gaganKangImg from '../static/images/gaganKang Small.png';
+import ivyWangImg from '../static/images/ivyWang Small.png';
+import jacksonFoltzImg from '../static/images/jacksonFoltz Small.png';
+import jasonZhangImg from '../static/images/jasonZhang Small.png';
+import jessicaHungImg from '../static/images/jessicaHung Small.png';
+import johnBaekImg from '../static/images/johnBaek Small.png';
+import kevinChuImg from '../static/images/kevinChu Small.png';
+import nickRodriguezImg from '../static/images/nickRodriguez Small.png';
+import sambaDiaImg from '../static/images/sambaDia Small.png';
+import sarinnaSungImg from '../static/images/sarinnaSung Small.png';
+import seanChungImg from '../static/images/seanChung Small.png';
+import steffenCornwellImg from '../static/images/steffenCornwell Small.png';
+import trangNguyenImg from '../static/images/trangNguyen Small.png';
+import yingTangImg from '../static/images/yingTang Small.png';
+
+const AboutTeam = () => {
+
+  return (
+    <div>
+      <Helmet>
+        <meta name="description" content="Keep.id" />
+      </Helmet>
+      <div className="bg-our-team-cover-img viewport-height-100 negative-margin-top">
+        <div className="container">
+          <div className="viewport-height-100 custom-vertical-center">
+            <div>
+              <h1 className="hero-header pb-4 text-white">
+              We are Team Keep.
+              </h1>
+              <p className="hero-header-2 text-white">
+                <img
+                  className="svg-white mb-5"
+                  src={RectangleSVG}
+                  alt="rectangle"
+                />
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <h1 className = "hero-header-aboutus text-black pb-3 ">Our board</h1>
+      <div className="profile-sizing profile-body">
+        <div className="grid">
+          <div className = "wrapper">
+            <ProfileCard
+              imgUrl = {ConnorChongImg}
+              title = {"Connor Chong"}
+              description={"Founder | CEO"}
+              info= {"#connor"}
+              id={"connor"}
+              infoadded={"my name is connor"}
```

Comment:
Just curious what is infoadded? and can we make it camelCase

Reply:
Changed to personDescription